src/components/menu/Menu.js

```diff
@@ -0,0 +1,100 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import { getDishById } from './helpers/menuHelper';
+import _omit from 'lodash/omit';
+
+//Constants
+import { INITIAL_STATE, CHECKOUT } from './constants/menu.general';
+
+//Components
+import MenuLinks from './components/menuLinks';
+import Dishes from './components/dishes';
+import Cart from './components/cart';
+
+//CSS
+import './menu.css';
+
+class Menu extends React.Component {
+  state = INITIAL_STATE;
+
+  resetCartItems = () => {
+    this.setState({
+      quantities: {},
```

Comment:
quantities state name doesn't implies what quantity you're keeping track of and in your case there's a mapping between quantity and ID so think of an alternate name